challengeMk2/DataBase/DatabaseConvertor.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using ChallengeMk2.Models;
+
+namespace ChallengeMk2.DataBase
+{
+    public class DatabaseConvertor

```

Comment:
Convertor or Converter ? 
Also, it looks more like a Mapper. Why do you need a mapper ?

Reply:
You're right, it is a mapper. I will rename it.
It is needed because the API send a JSON that contains properties  of custom types. SQlite table don't accept custom type so I use an "intermediary" class that contains all properties from the json but without custom type and put that in the db.